src/test/unit/controllers/SubscriptionSearchUrnResultController.test.ts

```diff
@@ -0,0 +1,57 @@
+import sinon from 'sinon';
+import { Request, Response } from 'express';
+import {PipApi} from '../../../main/utils/PipApi';
+import fs from 'fs';
+import path from 'path';
+import SubscriptionSearchUrnResultController from '../../../main/controllers/SubscriptionSearchUrnResultController';
+
+const axios = require('axios');
+jest.mock('axios');
+
+
+const api = new PipApi(axios);
+const subscriptionSearchUrnResultController = new SubscriptionSearchUrnResultController(api);
+const stub = sinon.stub(api, 'getSubscriptionByUrn');
+
+describe('Subscription Search Urn Result Controller', () => {
+  it('should render the urn search result page', () => {
+
+
+    const rawData = fs.readFileSync(path.resolve(__dirname, '../../../main/resources/mocks/subscriptionListResult.json'), 'utf-8');
+    const hearingsData = JSON.parse(rawData);
```

Comment:
need to change variable name here as well.

Reply:
done